start.sh

```diff
@@ -52,15 +53,26 @@ ssh sherlock mkdir -p $SHERLOCK_HOME/forward-util
 echo "== Uploading sbatch script =="
 scp sbatches/$SBATCH sherlock:$SHERLOCK_HOME/forward-util/
 
+# Give them one gpu :)
+if [ "${PARTITION}" == "gpu" ];
+  then
+      echo "== Requesting GPU =="
+      PARTITION="${PARTITION} --gres gpu:1"
```

Comment:
I like this!  Eventually, might be good to have setup.sh allow for specifying additional sbatch flags, or something like that.  But for now this is pretty good :)

Reply:
Agreed! After making this app --> https://researchapps.github.io/job-maker/ (try selecting "Sherlock" and then a parition, and look at the right side to see the sheer number of features to select from) I'm also astounded at the number of things that can be given to --gres. Maybe at some point I can make an associated little web interface to generate the parameters / start in some other way (but not for this PR!)